api/src/main/java/marquez/api/JobResource.java

```diff
@@ -213,6 +226,28 @@ void throwIfNotExists(@NonNull RunId runId) throws MarquezServiceException {
     }
   }
 
+  private void throwIfJobDoesNotMatchRun(RunId runId, String namespaceName, String jobName) {
+    Optional<Run> runRow = runService.getRun(runId);
```

Comment:
_Minor_: Here we are interactive with a `Run` object instead of a row and can adjust the naming to reflect that.